src/cluecode/finder.py

```diff
@@ -59,6 +59,23 @@ def logger_debug(*args):
 """
 
 
+def check_if_line_can_contain_urls(line):
```

Comment:
What about a shorter name? may be `contains_url`?

Reply:
Addressed your comment. :)